{{ cookiecutter.project_name }}/src/server.py

```diff
@@ -1,13 +1,16 @@
 import os
 
-from flows import example_flow
+from dotenv import load_dotenv
+from flows import main
 from prefect import serve
 
+load_dotenv()
+
 if __name__ == "__main__":
     assert "PREFECT_API_URL" in os.environ
     print(f"Serving flows for {os.environ['PREFECT_API_URL']}")
```

Comment:
serve() hoef je normaliter niet voor testen te doen, dat is alleen evt om te testen of je de syntax van de deploy goed hebt lijkt me. Je zal meestal de flow direct aanroepen (of een task testen met pytest).

=> even handmatig een environment variabele zetten lijkt me genoeg? Of we zetten er een default `localhost:4220/api` in.